src/sentry_check.py

```diff
@@ -520,3 +530,9 @@ def adapt_error(cls, e):
     "sentry.utils.compat import zip as izip`` instead."
 )
 B316.names = {"izip"}
+
+B317 = Error(
+    message=u"B317: json utilities should be imported from ``sentry.utils.json``."
```

Comment:
Is this actually implemented yet? In any case, parent module should be `sentry.utils.compat` for consistency. And so the error message should be `B316: Use ``from sentry.utils.compat import json`` instead.`

Reply:
We already have `sentry.utils.json`, it's used in a lot of places.

Reply:
But yeah agree on the message.

Reply:
Ah, so... mass-convert to `sentry.utils.compat.json` and then re-request review here?

Reply:
Yeah I'm almost done with that, we'll have to bump the version anyway.

Do you know how to deploy this to pypi

Reply:
Travis will do it. Push a commit with tag `0.3.2`.

Reply:
Added checks for `JSONEncoder`, `JSONDecodeError`, and `_default_encoder`.

I didn't add one for `JSONDecoder` since we don't export one.

Also I didn't get _too_ fancy with this message, since it changes a bit depending on style. I think this is fine.